doc/source/build-actions/index.rst

```diff
@@ -28,6 +28,38 @@ Here is a code sample for using this action:
             library-name: "ansys-<product>-<library>"
 
 
+here is a code for using this action
+
+Build wheelhouse action
+-----------------------
+This action builds the wheelhouse for a Python library and publishes them as
+artifacts.
+
+.. jinja:: build-wheelhouse
+
+    {{ inputs_table }}
+
+Here is a code sample for using this action:
+
+.. code-block:: yaml
+
+    build-wheelhouse:
+      name: Build the wheelhouse of the Python library
+      runs-on: ${{ matrix.os }}
+      strategy:
+         matrix:
+             os: [ubuntu-latest, windows-latest]
+             python-version: ['3.7', '3.8', '3.9', '3.10']
+      steps:
+        - name: "Build a wheelhouse of the Python library"
+          uses: pyansys/actions/build-wheelhouse@main
+          with:
+            library-name: "<ansys-product-library>"
+            library-namespace: "<ansys.product.libray>"
+            operating-system: ${{ matrix.os }}
+            python-version: ${{ matrix.python-version }}
```

Comment:
Why not just merge this action into a single call and provide the OS and Python versions as lists?

I guess that'd make it easier to use a plug-and-play stage of our workflow.

Reply:
It is a great idea! Let me check if reusable workflows support matrix strategy, not sure about this...

Reply:
Let me know if it does! Sounds interesting =)

Reply:
It is possible to call a composite workflow inside a matrix strategy. However, it is not possible to execute a matrix inside a composite workflow.

Reply:
Damn... that's a shame =( thanks for looking into it anyway! =)